pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -66,15 +67,19 @@ func TestProcessTicks(t *testing.T) {
 		Host:   "localhost",
 	}
 
+	cacheServ := &datasources.FakeCacheService{}
+	evaluator := eval.NewEvaluatorFactory(setting.UnifiedAlertingSettings{}, cacheServ, expr.ProvideService(&setting.Cfg{ExpressionsEnabled: true}, nil, nil, &featuremgmt.FeatureManager{}, nil, tracing.InitializeTracerForTest()), &pluginstore.FakePluginStore{})
```

Comment:
This test relied on there being no evaluation attempts. The guard prevents this now, so we must add an evaluator - a default, fake will do.